Fabric.Authorization.API/Modules/UsersModule.cs

```diff
@@ -134,7 +134,7 @@ private async Task<dynamic> GetUserPermissions(dynamic param)
                 Grain = userPermissionRequest.Grain,
                 SecurableItem = userPermissionRequest.SecurableItem,
                 IncludeSharedPermissions = isGrainEmpty,
-                UserGroups = groups?.Select(g => g.Name)
+                UserGroups = groups

```

Comment:
Do we need a similar change here: https://github.com/HealthCatalyst/Fabric.Authorization/pull/389/files#diff-8aa3fecd22b341d99ca92ccc12c0f59dR159, or is that already handled?

Reply:
Not sure what you mean by "similar change", but this endpoint should be handled. I switched the `PermissionResolutionRequest` from having a list of group names to having a list of `GroupIdentifier` objects.

Reply:
Just wanting to make sure the GetCurrentUserPermissions has been updated to reflect UserGroups changing from being a list of `string` to a list of `GroupIdentifier` objects.

Reply:
Ah yeah, it has been.